### ISSUE TITLE
Silliness

### ISSUE THREAD

**Issue by ericelliott** [sjsyrek/maryamyriameliamurphies.js#3]
Stuff like this:

```js
return m === Nothing ? true : false;
```

...can be shortened to this:

```js
return m === Nothing
```

**Comment by sjsyrek** [comment]
Yes, I have to go through and fix a bunch of things like that. They are generally artifacts of my direct translation of the Haskell code. I caught most of them in lazy-linked-lists. When I update this library, I'll certainly do the same. Thanks for taking the time to look at my code! :)

**Comment by geronimo-iia** [comment]
Hi, 
you have a pull request for this change https://github.com/sjsyrek/maryamyriameliamurphies.js/pull/9

**Comment by geronimo-iia** [comment]
nice stuff !!
Thanks for your work
